sdk/js/src/utils/aptos.ts

```diff
@@ -193,3 +229,44 @@ export async function getTypeFromExternalAddress(
     return null;
   }
 }
+
+export const getModuleAddressFromFullyQualifiedType = (
```

Comment:
discussed and we can change this to "coalesce" and remove the `isValidAptosType` requirement